### PR TITLE
Guidelines update april 24

### PR DESCRIPTION
**Pull request by rachaelbradley**
This is a pull request to publish the work on exploratory guidelines in autumn 2024-winter 2025 into the editor's draft.

The updated sections are:
- 1 Introduction - What’s new in this version of WCAG 3? 
- Guidelines

[Preview on Raw.git](https://raw.githack.com/w3c/wcag3/guidelines_update_april_24/guidelines/index.html)

[Google Doc Version](https://docs.google.com/document/d/1F_aA3yZHZR-FNKb9Vel8bQaiy1_CuT2L2qbf-UPG86o/edit?usp=sharing)

### CODE REVIEW

File: guidelines/index.html [hunk 1]
```diff
@@ -88,103 +89,801 @@ <h2>Guidelines</h2>
...
+
+			<ul><li>What outcomes are needed to make web content accessible and are missing from this list?</li>
+				<li>What research do you know of that support or refute these outcomes?</li>
+			</ul>
+
+			<p>We did not make changes to conformance related sections and we did not publish an updated WCAG 3 Explainer. </p>
+
+			<p id="additional_research"><strong>Additional Research</strong></p>
+			<p>One goal of publishing this list is to identify gaps in current research and request assistance filling those gaps.</p>
+			<p> Editor's notes indicate the outcomes within this list where the working group has not found enough research to fully validate the guidance and create methods to support it.  If you know of existing research or if you are interested in conducting research in this area, please file a GitHub issue or send email to <a href="mailto:public-agwg-comments@w3.org?subject=WCAG%203.0%20public%20comment">public-agwg-comments@w3.org</a> (<a href="https://lists.w3.org/Archives/Public/public-agwg-comments/">comment archive</a>). </p>
```
Comment (alastc) [inline guidelines/index.html:107]:
```suggestion
			<p> Editor's notes indicate the outcomes within this list where the working group has not found enough research to fully validate the guidance and create methods to support it.  If you know of existing research or if you are interested in conducting research in this area, please file a <a href="https://github.com/w3c/wcag3/issues">GitHub issue</a> or send email to <a href="mailto:public-agwg-comments@w3.org?subject=WCAG%203.0%20public%20comment">public-agwg-comments@w3.org</a> (<a href="https://lists.w3.org/Archives/Public/public-agwg-comments/">comment archive</a>). </p>
```

**Comment by rachaelbradley** [comment]
> I can't find it in the source, but there is a plain language section at the top of the "Guidelines" section:
> 
> > Plain language summary of "Guidelines"
> > The following guidelines are being considered for WCAG 3. They are currently a list of topics which we expect to explore more thoroughly in future drafts. The list includes current WCAG 2 guidance.
> 
> I suggest:
> 
> > The following guidelines are being considered for WCAG 3. They are currently a list of topics which we expect to explore more thoroughly in future drafts. The list includes current WCAG 2 guidance **and some additional requirements**.

I've made this change.

**Comment by julierawe** [comment]
Thank you for alphabetizing the outcomes in each section—it helped me notice two subgroups had submitted similar-sounding outcomes in "Text and wording." 

Can we change "Verbosity" to "Interface verbosity" so it's clearer that this outcome is different from "Unnecessary words or phrases," which is plain-language guidance that applies to all sentences?

**Comment by bruce-usab** [comment]
I love every little thing except "aggregate".  Some minor editorial in the Google Doc.

File: guidelines/index.html [hunk 2]
```diff
@@ -81,110 +86,814 @@ <h3>Filtering less mature content</h3>
...
 
-				<p>The following guidelines are an initial list that the working group will be exploring. <strong>The following guidelines are initial groupings to guide the next phase of work. They should be considered drafts and should not be considered as final content of WCAG 3</strong>.</p>
+				<p><strong> The following guidelines are an initial list of potential outcomes that the working group will be exploring. The goal is to guide the next phase of work. They should be considered drafts and should not be considered as final content of WCAG 3</strong>.</p>
+
+			<p>The outcomes listed below are have not been refined and do not include essential exceptions or methods. Some outcomes may be best addressed by authoring tools or at the platform level. We will address these questions as we further explore each outcome.  </p>
```
Comment (mattgarrish) [inline guidelines/index.html:98]:
```suggestion
			<p>The outcomes listed below have not been refined and do not include essential exceptions or methods. Some outcomes may be best addressed by authoring tools or at the platform level. We will address these questions as we further explore each outcome.  </p>
```

**Comment by alastc** [comment]
Closing, replaced by #63

**Pull request closed without merging**